### PR TITLE
[PREVIEW] Feature/sl 1971/list hearings session util

### PR DESCRIPTION
**Pull request by radoslawlandowskikainos**

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sandl/snlevents/repository/queries/HearingForListingQueries.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    private final String hearingForListingSelectCount = "SELECT Count(*) ";
+    private final String hearingForListingSelectHearings = "SELECT h.id, case_number, case_title, "
+        + "ct.code as case_type_code, "
+        + "ct.description as case_type_description, ht.code as hearing_type_code, "
+        + "ht.description as hearing_type_description, (duration * 1000000000) as duration, "
+        + "schedule_start, schedule_end, version, "
+        + "priority, communication_facilitator, reserved_judge_id, p.name as reserved_judge_name, number_of_sessions, "
+        + "h.status, is_multisession ";
+
+    private final String hearingForListingQueryBody = "FROM hearing h "
```
Comment (zielinskikamil) [inline src/main/java/uk/gov/hmcts/reform/sandl/snlevents/repository/queries/HearingForListingQueries.java]: why not merge them into one constant?

File: src/main/java/uk/gov/hmcts/reform/sandl/snlevents/repository/queries/HearingForListingQueries.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        + "LEFT JOIN person p on p.id = h.reserved_judge_id "
+        + "INNER JOIN case_type ct on h.case_type_code = ct.code "
+        + "INNER JOIN hearing_type ht on h.hearing_type_code = ht.code "
+        + "INNER JOIN status_config sc on h.status = sc.status "
+        + "WHERE can_be_listed = true "
+        + "AND h.status != 'Listed' "
+        + "AND is_deleted = false ";
```
Comment (zielinskikamil) [inline src/main/java/uk/gov/hmcts/reform/sandl/snlevents/repository/queries/HearingForListingQueries.java]: its minor but I would recommend writing all SQL syntax with capitals characters (ON, AS, TRUE)

File: src/main/java/uk/gov/hmcts/reform/sandl/snlevents/repository/queries/HearingForListingQueries.java (hunk 1 above)
Comment (radoslawlandowskikainos) [reply src/main/java/uk/gov/hmcts/reform/sandl/snlevents/repository/queries/HearingForListingQueries.java]: It's used in 'totalcount' query and 'entities' query

**Comment by radoslawlandowskikainos** [comment]
Closing this PR due to jenkins failures. New one will be reopened.

**Pull request closed without merging**